tags/cncf-tags-github-org.md

```diff
@@ -0,0 +1,66 @@
+# `cncf-tags` GitHub org
+
+The [`cncf-tags` GitHub org](https://github.com/cncf-tags) serves as a home for TAG-sponsored projects and tools.
+This org is intended to provide a vendor-neutral place for TAGs to collaborate on projects
+endorsed by and actively worked on by members of a TAG.
+The org will _not_ contain projects that can potentially apply to become CNCF projects.
+
+Each repo has a "topic" of the form `tag-<tag-name>`. This ensures that repos are easily searchable per TAG
+through repo topics. For example, all repos under tag-runtime can be found at https://github.com/topics/tag-runtime.
+
+## Creating a new repo
+
+To request for a new repo, an issue must be created against the `cncf/toc` repo.
```

Comment:
I guess this should be by using the new template above? If so, let's link it!

Reply:
> I guess this should be by using the new template above? If so, let's link it!

Good point, done 👍